lib/foreman_puppet_enc/engine.rb

```diff
@@ -64,6 +64,13 @@ class Engine < ::Rails::Engine
         divider :top_menu, parent: :configure_menu, caption: N_('Puppet ENC')
 
         # add menu entries
+
+        add_menu_item :top_menu, :puppet_enc, {
+          caption: 'Config Groups',
+          engine: ForemanPuppetEnc::Engine, parent: :configure_menu, after: :ansible,
+          url_hash: { controller: 'foreman_puppet_enc/config_groups', action: :index }
+        }
```

Comment:
This doesn't belong here I belive :)
I've renamed the menu a bit, next lines are correct I belive :)